src/quo2/components/drawers/drawer_buttons/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.drawers.drawer-buttons.view
+  (:require [react-native.core :as rn]
+            [quo2.components.icon :as icon]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.drawers.drawer-buttons.style :as style]))
+
+(defn render-bottom
+  [children]
+  [rn/view
+   {:flex-direction  :row
+    :justify-content :space-between}
+   [children]
+   [rn/view
+    {:style {:border-radius   40
+             :border-width    1
+             :margin-left     24
+             :height          28
+             :width           28
+             :justify-content :center
+             :align-items     :center
+             :border-color    (colors/alpha colors/white 0.05)}}
+    [icon/icon :arrow-right
+     {:color colors/white
+
+      :size  20}]]])
+
+(defn render-children-bottom
+  [children]
+  (if (or (string? children) (keyword children))
```

Comment:
render subtitle/children can be both a string or a render prop as the text examples used require inline styling on certain words in the text.